hoot-core/src/main/cpp/hoot/core/algorithms/MultiLineStringSplitter.h

```diff
@@ -28,7 +28,7 @@
 #define MULTILINESTRINGSPLITTER_H
 
 #include <hoot/core/OsmMap.h>
-#include <hoot/core/algorithms/linearreference/WaySublineString.h>
+#include <hoot/core/algorithms/linearreference/WaySublineCollection.h>
```

Comment:
Don't forget to update comments w/ the refactor of WaySublineString->WaySublineCollection !